### PR TITLE
[Term Entry] Java Calendar Class Method: .getDisplayName()

### LINKED ISSUE #2979: [Term Entry] Java Calendar getDisplayName()
### Reason/inspiration (optional)

> **Note:** Different from the `getDisplayNames()` method, there is a separate issue for `getDisplayNames()` method.

Please add the `getDisplayName()` method as a new term entry for the [Calendar](https://www.codecademy.com/resources/docs/java/calendar) class. Provide the entry in a markdown file and place under the file path `content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md`

The entry should have the following:

–  An introductory paragraph about what the method is and does.
–  A syntax section with a pseudo code block with an explanation of it.
–  An example showing the method in use.

Please refer to the [term entry templates](https://github.com/Codecademy/docs/blob/main/documentation/term-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by DakuwoN**
### Description

- Added term entry for the Java Calendar Class method getDisplayName
- Closes #2979 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [ ] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
+---
+Title: '.getDisplayName()' 
+Description: 'Returns the string representation (display name) of the calender field value in the given style and locale. If no string representation is applicable, null is returned.' 
+Subjects: 
+  - 'Code Foundations'
+  - 'Computer Science'
+
```
Comment (ishg-153) [inline content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md:7]:
```suggestion
  - 'Computer Science'
```

File: content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+  - 'Date'
+  - 'Methods'
+  - 'Calender'
+CatalogContent: 
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+
+The **.getDisplayName()** method of the `Calender` class returns the string representation (display name) of the calender field value in the given style and locale. If no string representation is applicable, null is returned.
```
Comment (ishg-153) [inline content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md:18]:
```suggestion
The **`.getDisplayName()`** method of the `Calender` class returns the string representation (display name) of the calender field value in the given style and locale. If no string representation is applicable, null is returned.
```

File: content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+```pseudo
+// Create a Calendar object.
+Calendar calendar = Calendar.getInstance();
+
+// Get the display name for the month in the current locale.
+String monthName = calendar.getDisplayName(Calendar.MONTH, Calendar.SHORT, Locale.getDefault());
+
+// Print the month name.
+System.out.println(monthName);
+```
```
Comment (ishg-153) [inline content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md:35]: This part isn't necessary. Syntax will only consist of how the method would look in a code.

File: content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+  - 'paths/computer-science'
+---
+
+
+The **.getDisplayName()** method of the `Calender` class returns the string representation (display name) of the calender field value in the given style and locale. If no string representation is applicable, null is returned.
+
+## Syntax
+
+```pseudo
+public String getDisplayName(int field, int style, Locale locale)
```
Comment (ishg-153) [inline content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md:23]:
```suggestion
calendarInstance.getDisplayName(calendarfield, calendarstyle, locale)
```

File: content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+
+// Get the display name for the month in the current locale.
+String monthName = calendar.getDisplayName(Calendar.MONTH, Calendar.SHORT, Locale.getDefault());
+
+// Print the month name.
+System.out.println(monthName);
+```
+
+## Parameters
+
```
Comment (ishg-153) [inline content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md:38]:
```suggestion
```

File: content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+- `field`: The first parameter field, specifies the calender field for which the display name is returned. Possible values for the field parameter include: `Calender.MONTH`, `Calender.DAY_OF_WEEK`, `Calender.YEAR`.
+
+- `style`: The second parameter, style, specifies the style of the display name. Possible values for the style parameter include: `Calender.SHORT` and `Calender.LONG`. Both values can either return a short or long form of the display name.
+
+- `locale`: The third parameter, locale, specifies the locale for which the display name is returned. Possible values for the field parameter include: `Locale.ENGLISH`, `Locale.FRENCH`, `Locale.GERMAN`.
```
Comment (ishg-153) [inline content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md:43]: Here, you should use the variables, and parameters from syntax to further explain them. For example:
- `calendarInstance`: Instance of a calendar

File: content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+    // Get the display name for the month in the current locale.
+    String monthName = calendar.getDisplayName(Calendar.MONTH, Calendar.SHORT, Locale.getDefault());
+
+    // Print the month name.
+    System.out.println(monthName);
+  }
+}
+```
+
+This code will print the name of the current month in the default locale. For example, if the default locale is English, the code will print "January".
```
Comment (ishg-153) [inline content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md:66]:
```suggestion
This code will print the name of the current month in the default locale.
```

Follow this with a ` ```shell ` block that displays the output of the above example.

File: content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md [hunk 8]
```diff
@@ -0,0 +1,87 @@
...
+    // Get the display name for the month in the French locale.
+    String monthName = calendar.getDisplayName(Calendar.MONTH, Calendar.LONG, Locale.FRENCH);
+
+    // Print the month name.
+    System.out.println(monthName);
+  }
+}
+```
+
+Here is another example of how to use the getDisplayName() method with three parameters. The code will print the name of the current month in the French locale. For example, if the current locale is French, the code will print "janvier".
```
Comment (ishg-153) [inline content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md:87]:
```suggestion
Here is another example of how to use the getDisplayName() method with three parameters. The code will print the name of the current month in the French locale.
```

Again here add the output of the example in a ` ``` shell ` block.

**Review by ishg-153 — CHANGES_REQUESTED** [review]
@DakuwoN I have suggested some changes, please address them.

File: content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md (hunk 1 above)
Comment (DakuwoN) [reply content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md:7]: I am not understanding the change here? Sorry, I am a newbie! Is it the spacing? And hey man I just realized I can't spell to save my life! I fixed the word 'calendar' ... as I had all the words spelled... 'calender' HAHHAHA, I seen you wrote it correctly as well on the Syntax header :D ty

File: content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md (hunk 3 above)
Comment (DakuwoN) [reply content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md:35]: Oopps sorry, thank you.

File: content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md (hunk 5 above)
Comment (DakuwoN) [reply content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md:38]: Again, I am not sure of the change here?

File: content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md (hunk 6 above)
Comment (DakuwoN) [reply content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md:43]: Sorry about that, working on it now.

File: content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md (hunk 1 above)
Comment (ishg-153) [reply content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md:7]: Yes that's correct I just removed the extra line after computer science.

File: content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md (hunk 5 above)
Comment (ishg-153) [reply content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md:38]: Removed the parameter heading. It is expected that the syntax subsection will have parameter explanation.

**Review by ishg-153 — COMMENTED** [review]
Once you are done resolving all the suggestions you can tag me or request a review. Then we can move forward.

**Comment by DakuwoN** [comment]
@ishg-153 Tried my best to fix the errors, thank you!

File: content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md [hunk 9]
```diff
@@ -0,0 +1,89 @@
+---
+Title: '.getDisplayName()' 
+Description: 'Returns the string representation (display name) of the calendar field value in the given style and locale. If no string representation is applicable, null is returned.' 
+Subjects: 
+  - 'Code Foundations'
+  - 'Computer Science'
+ Tags:
```
Comment (ishg-153) [inline content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md:7]:
```suggestion
Tags: 
```

File: content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md [hunk 10]
```diff
@@ -0,0 +1,89 @@
...
+calendarInstance.getDisplayName(calendarField, calendarStyle, locale)
+```
+
+- `calendarInstance`: An instance of a calendar.
+
+- `calendarField`: The first parameter field, specifies the calendar field for which the display name is returned. Possible values for the field parameter include: `Calendar.MONTH`, `Calendar.DAY_OF_WEEK`, `Calendar.YEAR`.
+
+- `calendarStyle`: The second parameter, style, specifies the style in which the display name should be presented. Possible values for the style parameter include: `Calendar.SHORT` and `Calendar.LONG`. Both values can either return a short or long form of the display name.
+
+- `locale`: The third parameter, locale, specifies the locale in which to display the name. Possible values for the field parameter include: `Locale.US`, `Locale.FRANCE`, `Locale.GERMANY`.
```
Comment (ishg-153) [inline content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md:31]:
```suggestion
- `locale`: The third parameter, locale, specifies the locale in which to display the name. Possible values for the field parameter include: `Locale.US`, `Locale.FRANCE`, `Locale.GERMANY`, etc.
```

File: content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md [hunk 11]
```diff
@@ -0,0 +1,89 @@
...
+  }
+}
+```
+
+This code will print the name of the current month in the default locale.
+
+The output for the code above:
+
+```shell
+"January"
```
Comment (ishg-153) [inline content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md:59]:
```suggestion
Oct
```

Change the month to October that way- it is easy to verify the code. Also, as it uses `.SHORT`, the displayed month will be short. The shell block should always show the code output as it is so, no double quotes.

File: content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md [hunk 12]
```diff
@@ -0,0 +1,89 @@
...
+  }
+}
+```
+
+Here is another example of how to use the getDisplayName() method with three parameters. The code will print the name of the current month in the French locale.
+
+Output for the code above:
+
+```shell
+"Janvier"
```
Comment (ishg-153) [inline content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md:88]:
```suggestion
octobre
```

Same here.

File: content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md [hunk 13]
```diff
@@ -0,0 +1,89 @@
...
+
+## Syntax
+
+```pseudo
+calendarInstance.getDisplayName(calendarField, calendarStyle, locale)
+```
+
+- `calendarInstance`: An instance of a calendar.
+
+- `calendarField`: The first parameter field, specifies the calendar field for which the display name is returned. Possible values for the field parameter include: `Calendar.MONTH`, `Calendar.DAY_OF_WEEK`, `Calendar.YEAR`.
```
Comment (ishg-153) [inline content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md:27]:
```suggestion
- `calendarField`: The first parameter field, specifies the calendar field for which the display name is returned. Possible values for the field parameter include: `Calendar.MONTH`, `Calendar.DAY_OF_WEEK`, and `Calendar.YEAR`.
```

File: content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md [hunk 14]
```diff
@@ -0,0 +1,89 @@
...
+
+- `calendarStyle`: The second parameter, style, specifies the style in which the display name should be presented. Possible values for the style parameter include: `Calendar.SHORT` and `Calendar.LONG`. Both values can either return a short or long form of the display name.
```
Comment (ishg-153) [inline content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md:29]:
```suggestion
- `calendarStyle`: The second parameter, style, specifies the style in which the display name should be presented. Possible values for the style parameter include: `Calendar.SHORT`, and `Calendar.LONG`. They can return a short and long form of the display name, respectively.
```

**Review by ishg-153 — CHANGES_REQUESTED** [review]
@DakuwoN there are a few more changes I have suggested, please address them.

File: content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md (hunk 9 above)
Comment (DakuwoN) [reply content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md:7]: Oops sorry, fixed.

File: content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md (hunk 10 above)
Comment (DakuwoN) [reply content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md:31]: Got it. ty

File: content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md (hunk 11 above)
Comment (DakuwoN) [reply content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md:59]: Sorry, yeah that makes sense haha

File: content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md (hunk 12 above)
Comment (DakuwoN) [reply content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md:88]: Roger that

File: content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md (hunk 13 above)
Comment (DakuwoN) [reply content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md:27]: got it

File: content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md (hunk 14 above)
Comment (DakuwoN) [reply content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md:29]: ty

**Comment by DakuwoN** [comment]
Thank you so much for helping me @ishg-153 and being patient with me! That was so fun, I love doing this, I am going to try and learn some more! 👯

File: content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md [hunk 15]
```diff
@@ -0,0 +1,81 @@
...
+}
+```
+
+This code will print the name of the current month in the default locale. The output for the code above:
+
+```shell
+Oct
+```
+
+----
```
Comment (SSwiniarski) [inline content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md:56]:
```suggestion
Here is another example of how to use the getDisplayName() method with three parameters. The code will print the name of the current month in the French locale.
```

File: content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md [hunk 16]
```diff
@@ -0,0 +1,81 @@
...
+    // Get the display name for the month in the French locale.
+    String monthName = calendar.getDisplayName(Calendar.MONTH, Calendar.LONG, Locale.FRENCH);
+
+    // Print the month name.
+    System.out.println(monthName);
+  }
+}
+```
+
+Here is another example of how to use the getDisplayName() method with three parameters. The code will print the name of the current month in the French locale. Output for the code above:
```
Comment (SSwiniarski) [inline content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md:77]:
```suggestion
Output for the code above:
```

File: content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md [hunk 17]
```diff
@@ -0,0 +1,81 @@
...
+    // Get the display name for the month in the current locale.
+    String monthName = calendar.getDisplayName(Calendar.MONTH, Calendar.SHORT, Locale.getDefault());
+
+    // Print the month name.
+    System.out.println(monthName);
+  }
+}
+```
+
+This code will print the name of the current month in the default locale. The output for the code above:
```
Comment (SSwiniarski) [inline content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md:50]:
```suggestion
The output for the code above:
```

File: content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md [hunk 18]
```diff
@@ -0,0 +1,81 @@
...
+calendarInstance.getDisplayName(calendarField, calendarStyle, locale)
+```
+
+- `calendarInstance`: An instance of a calendar.
+- `calendarField`: The first parameter field, specifies the calendar field for which the display name is returned. Possible values for the field parameter include: `Calendar.MONTH`, `Calendar.DAY_OF_WEEK`, and `Calendar.YEAR`.
+- `calendarStyle`: The second parameter, style, specifies the style in which the display name should be presented. Possible values for the style parameter include: `Calendar.SHORT`, and `Calendar.LONG`. They can return a short and long form of the display name, respectively.
+- `locale`: The third parameter, locale, specifies the locale to display the name. Possible values for the field parameter include: `Locale.US`, `Locale.FRANCE`, `Locale.GERMANY`, etc.
+
+## Example
+
```
Comment (SSwiniarski) [inline content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md:30]:
```suggestion

This code will print the name of the current month in the default locale.

```

File: content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md [hunk 19]
```diff
@@ -0,0 +1,81 @@
...
+```pseudo
+calendarInstance.getDisplayName(calendarField, calendarStyle, locale)
+```
+
+- `calendarInstance`: An instance of a calendar.
+- `calendarField`: The first parameter field, specifies the calendar field for which the display name is returned. Possible values for the field parameter include: `Calendar.MONTH`, `Calendar.DAY_OF_WEEK`, and `Calendar.YEAR`.
+- `calendarStyle`: The second parameter, style, specifies the style in which the display name should be presented. Possible values for the style parameter include: `Calendar.SHORT`, and `Calendar.LONG`. They can return a short and long form of the display name, respectively.
+- `locale`: The third parameter, locale, specifies the locale to display the name. Possible values for the field parameter include: `Locale.US`, `Locale.FRANCE`, `Locale.GERMANY`, etc.
+
+## Example
```
Comment (SSwiniarski) [inline content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md:29]:
```suggestion
## Examples
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@DakuwoN I did a second review and had a few changes.

File: content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md (hunk 19 above)
Comment (DakuwoN) [reply content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md:29]: Sorry, fixed.

File: content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md [hunk 20]
```diff
@@ -0,0 +1,81 @@
...
+    Calendar calendar = Calendar.getInstance();
+
+    // Get the display name for the month in the current locale.
+    String monthName = calendar.getDisplayName(Calendar.MONTH, Calendar.SHORT, Locale.getDefault());
+
+    // Print the month name.
+    System.out.println(monthName);
+  }
+}
+ The output for the code above:
```
Comment (SSwiniarski) [inline content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md:50]:
````suggestion
```

 The output for the code above:
````

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@DakuwoN One more comment.

File: content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md (hunk 20 above)
Comment (DakuwoN) [reply content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md:50]: Thanks @SSwiniarski I got it, I will fix tomorrow!

**Review by SSwiniarski — APPROVED** [review]
@DakuwoN Looks good.

**Pull request merged**